manifest/kubernetes/rhel/cluster-network-04-loadbalancers.vmware.com.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I think it's a good point. Wwe don't need cluster-network-xx-xx in native kubernetes and it can simplify the manifests generation. Will try it in the next patch. Thanks!